doc/user/bgp.rst

```diff
@@ -4109,6 +4109,124 @@ structure is extended with :clicmd:`show bgp [afi] [safi]`.
    If ``afi`` is specified, with ``all`` option, routes will be displayed for
    each SAFI in the selected AFI.
 
+.. clicmd:: show [ip] bgp [vrf VIEWVRFNAME$vrf_name] [afi] [safi] detail [json]
+
+   Display the detailed version of all routes from the specified bgp vrf table
+   for a given afi + safi.
+
+   If no vrf is specified, then it is assumed as a default vrf and routes
+   are displayed from default vrf table.
+
+   If ``all`` option is specified as vrf name, then all bgp vrf tables routes
+   from a given afi+safi are displayed in the detailed output of routes.
+
+   If ``json`` option is specified, detailed output is displayed in JSON format.
+
+   Following are sample output for few examples of how to use this command.
+
+.. code-block:: frr
+
+   torm-23# sh bgp ipv4 unicast detail (OR) sh bgp vrf default ipv4 unicast detail
+
+   !--- Output suppressed.
+
+   BGP routing table entry for 27.0.0.19/32
```

Comment:
Also, could you replace all public IP ranges to IPv4 documentation ranges? https://datatracker.ietf.org/doc/html/rfc5737

Reply:
done